apps/balancer-tools/src/app/apr/pool/[network]/[poolId]/page.tsx

```diff
@@ -3,30 +3,47 @@ import { Suspense } from "react";
 import ChartSkelton from "#/app/apr/(components)/(skeleton)/ChartSkelton";
 import KpisSkeleton from "#/app/apr/(components)/(skeleton)/KpisSkeleton";
 import TableSkeleton from "#/app/apr/(components)/(skeleton)/TableSkeleton";
-import PoolOverviewCards from "#/app/apr/pool/(components)/PoolOverviewCards";
+import { SearchParams } from "#/app/apr/page";
 import Breadcrumb from "#/app/apr/round/(components)/Breadcrumb";
 
 import HistoricalCharts from "../../(components)/HistoricalCharts";
+import PoolOverviewCards from "../../(components)/PoolOverviewCards";
 import PoolTokens from "../../(components)/PoolTokens";
 import { YieldWarning } from "../../(components)/YieldWarning";
 
 export default async function Page({
   params: { poolId },
+  searchParams,
 }: {
+  searchParams: SearchParams;
   params: { poolId: string };
 }) {
+  const {
+    startAt: startAtDate,
+    endAt: endAtDate,
+    // @ts-ignore
+  } = QueryParamsSchema.safeParse(searchParams).data;
```

Comment:
Error on typing
```
Property 'data' does not exist on type 'SafeParseReturnType<{ poolId?: string | null | undefined; startAt?: string | null | undefined; endAt?: string | null | undefined; sort?: string | null | undefined; order?: Order | undefined; limit?: string | ... 1 more ... | undefined; ... 9 more ...; maxTvl?: string | ... 1 more ... | undefined; }, { ...; }>'.
  Property 'data' does not exist on type 'SafeParseError<{ poolId?: string | null | undefined; startAt?: string | null | undefined; endAt?: string | null | undefined; sort?: string | null | undefined; order?: Order | undefined; limit?: string | ... 1 more ... | undefined; ... 9 more ...; maxTvl?: string | ... 1 more ... | undefined; }>'.ts(2339)
```

Reply:
if you're using safeParse, you first need to ensure the parse was successful: https://github.com/colinhacks/zod#safeparse